frontend/src/pages/Result.jsx

```diff
@@ -0,0 +1,514 @@
+              <div className="text-wrapper-10" style={setFont(1)} >{lpvsResult?.lpvsResultFileList[1]?.licenseSpdx}</div>
+              <div className="text-wrapper-11" style={setFont(0)} >{lpvsResult?.lpvsResultFileList[0]?.licenseSpdx}</div>
+              <div className="text-wrapper-12">Detected Licenses</div>
+            </div>
+            <div className="match-value-chart">
+              <div className="text-wrapper-13">Match Value</div>
+              <div className="text-wrapper-14">{lpvsResult?.lpvsResultFileList[4]?.matchValue}</div>
+              <div className="text-wrapper-15">{lpvsResult?.lpvsResultFileList[3]?.matchValue}</div>
+              <div className="text-wrapper-16">{lpvsResult?.lpvsResultFileList[2]?.matchValue}</div>
+              <div className="text-wrapper-17">{lpvsResult?.lpvsResultFileList[1]?.matchValue}</div>
+              <div className="text-wrapper-18">{lpvsResult?.lpvsResultFileList[0]?.matchValue}</div>
+            </div>
+            <div className="match-line-chart">
+              <div className="text-wrapper-19">{lpvsResult?.lpvsResultFileList[4]?.matchLine}</div>
+              <div className="text-wrapper-20">{lpvsResult?.lpvsResultFileList[3]?.matchLine}</div>
+              <div className="text-wrapper-21">{lpvsResult?.lpvsResultFileList[2]?.matchLine}</div>
+              <div className="text-wrapper-22">{lpvsResult?.lpvsResultFileList[1]?.matchLine}</div>
+              <div className="text-wrapper-23">{lpvsResult?.lpvsResultFileList[0]?.matchLine}</div>
+              <div className="text-wrapper-24">Match Line</div>
+            </div>
+            <div className="path-chart">
+              <div className="text-wrapper-25-container">
+                <div className={`text-wrapper-25 ${lpvsResult?.lpvsResultFileList[4]?.path.length >= 28 ? 'long' : 'short'}`}>{lpvsResult?.lpvsResultFileList[4]?.path}</div>
+              </div>
+              <div className="text-wrapper-26-container">
+                <div className={`text-wrapper-26 ${lpvsResult?.lpvsResultFileList[3]?.path.length >= 28 ? 'long' : 'short'}`}>{lpvsResult?.lpvsResultFileList[3]?.path}</div>
+              </div>
+              <div className="text-wrapper-27-container">
+                <div className={`text-wrapper-27 ${lpvsResult?.lpvsResultFileList[2]?.path.length >= 28 ? 'long' : 'short'}`}>{lpvsResult?.lpvsResultFileList[2]?.path}</div>
+              </div>
+              <div className="text-wrapper-28-container">
+                <div className={`text-wrapper-28 ${lpvsResult?.lpvsResultFileList[1]?.path.length >= 28 ? 'long' : 'short'}`}>{lpvsResult?.lpvsResultFileList[1]?.path}</div>
+              </div>
+              <div className="text-wrapper-29-container">
+                <div className={`text-wrapper-29 ${lpvsResult?.lpvsResultFileList[0]?.path.length >= 28 ? 'long' : 'short'}`}>{lpvsResult?.lpvsResultFileList[0]?.path}</div>
```

Comment:
If the `path` value is `NULL`, it goes to the error.

**TEST CASE** : https://github.com/tiokim/edge-home-orchestration-go/pull/100
**DB Records**
```
mysql> select * from detected_license where pull_request_id=11;
+----+-----------------+------------+-------------+-----------------------+----------------------------------------------+------------+-------------+-------------+---------------------+----------------------------------------------------------------------+----------------+-----------------+---------------------------------------+-------------------+------------------+--------------+
| id | pull_request_id | license_id | conflict_id | repository_license_id | file_path                                    | match_type | match_value | match_lines | component_file_path | component_file_url                                                   | component_name | component_lines | component_url                         | component_version | component_vendor | issue        |
+----+-----------------+------------+-------------+-----------------------+----------------------------------------------+------------+-------------+-------------+---------------------+----------------------------------------------------------------------+----------------+-----------------+---------------------------------------+-------------------+------------------+--------------+
|  8 |              11 |          6 |        NULL |                  NULL | internal/controller/scoringmgr/scoringmgr.go | snippet    | 63%         | 53-145      | fs/exfat/misc.c     | https://osskb.org/api/file_contents/0e0cc80e43dc53f42483f6aa8e295f7b | cloud-kernel   | 1-93            | https://gitee.com/anolis/cloud-kernel | 5.19.0-1_rc1      | anolis           | 0x01         |
|  9 |              11 |          4 |        NULL |                  NULL | internal/controller/scoringmgr/scoringmgr.go | snippet    | 63%         | 53-145      | fs/exfat/misc.c     | https://osskb.org/api/file_contents/0e0cc80e43dc53f42483f6aa8e295f7b | cloud-kernel   | 1-93            | https://gitee.com/anolis/cloud-kernel | 5.19.0-1_rc1      | anolis           | 0x01         |
| 10 |              11 |       NULL |           1 |                     1 | NULL                                         | NULL       | NULL        | NULL        | NULL                | NULL                                                                 | NULL           | NULL            | NULL                                  | NULL              | NULL             | 0x01         |
+----+-----------------+------------+-------------+-----------------------+----------------------------------------------+------------+-------------+-------------+---------------------+----------------------------------------------------------------------+----------------+-----------------+---------------------------------------+-------------------+------------------+--------------+

```

Reply:
Fixed an error for NULL.